package.json

```diff
@@ -92,5 +92,8 @@
     "scoring",
     "entropy",
     "bruteforce"
-  ]
+  ],
+  "dependencies": {
+    "jszip": "^3.6.0"
```

Comment:
This needs to be in the devDependencies. It will never be executed by the library